prime-router/settings/organizations-prod.yml

```
@@ -280,7 +285,21 @@
         useBatchHeaders: true
       timing:
         operation: MERGE
-        numberPerDay: 12
+        numberPerDay: 1         # change to once a day for download site
+        initialTime: 01:15
```

```suggestion
        initialTime: 09:15
```

0915 Eastern would be the same timing as AZ and Pima, and would clear any early morning submissions for a potential 9:30am deploy.